web/src/lib/components/pages.tsx

```diff
@@ -242,31 +242,21 @@ export default class Pages extends Component<PagesProps, PagesState> {
                            progressCb: Function): Promise<void> {
     try {
       await this.ensurePrivacyAgreement();
-      let runningTotal = 0;
-      let originalTotal = recordings.length;
+      const originalTotal = recordings.length;
 
       // This function calls itself recursively until
       // all recordings are uploaded.
```

Comment:
minor: this code comment is no longer accurate ;)

Reply:
Okay, removed. Thanks for spotting!